src/Generators/DataTablesMakeCommand.php

```diff
@@ -128,6 +207,8 @@ protected function getOptions()
     {
         return [
             ['model', null, InputOption::VALUE_NONE, 'Use the provided name as the model.', null],
+            ['action', null, InputOption::VALUE_OPTIONAL, 'Force the use of singular in filename.', null],
```

Comment:
Maybe a better description?

`Force the use of singular in filename.` 

to something like:

`Path to action column template.`